pages/gallery.html

```diff
@@ -14,16 +14,58 @@
     <title>Galeria - Hostal las paisas</title>
   </head>
   <body>
-    <nav>
-      <!-- nav bar section -->
-    </nav>
     <section class="small-hero small-hero__image--gallery ">
       <div class="small-hero__layer"></div>
       <div class="small-hero__layer small-hero__layer--darker"></div>
       <h1 class="small-hero__title">Galería</h1>
     </section>
+    <nav>
+      <!-- nav bar section -->
+    </nav>
+    <!-- gallery -->
+    <section class="container container-componet info-gallery">
+      <div class="info-gallery__text">    
+        <p class="text-p">Nuestro hostal te ofrece todas las comodidades para que vivas la mejor experiencia. Observa todo lo que tenemos para ti en el majestuoso paraíso de Capurganá</p>
+      </div>
+      <div class="container-images" id="divImagesContainer">
+        <div class="container-images__item">
+          <div class="item__images"></div>
+          <div class="item--overlay"><img src="./../assets/images/gallery/expand_icon.svg" class="overlay" alt=""></div>
+        </div>
+        <div class="container-images__item">
+          <div class="item__images"></div>
+          <div class="item--overlay"><img src="./../assets/images/gallery/expand_icon.svg" class="overlay" alt=""></div>
+        </div>
+        <div class="container-images__item">
+          <div class="item__images"></div>
+          <div class="item--overlay"><img src="./../assets/images/gallery/expand_icon.svg" class="overlay" alt=""></div>
+        </div>
+        <div class="container-images__item">
+          <div class="item__images"></div>
+          <div class="item--overlay"><img src="./../assets/images/gallery/expand_icon.svg" class="overlay" alt=""></div>
+        </div>
+        <div class="container-images__item">
+          <div class="item__images"></div>
+          <div class="item--overlay"><img src="./../assets/images/gallery/expand_icon.svg" class="overlay" alt=""></div>
+        </div>
+        <div class="container-images__item">
+          <div class="item__images"></div>
```

Comment:
maybe this class could be `item__image` instead of `item__images` since it represents a single image